pep/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2017",
+    "target": "es2020",
```

Comment:
any particular reason you're bumping the compile target up to ES2020? not specifically against it and I know we targeting mostly evergreen browsers in this app, but just want to make sure we don't cause any browser incompatibility issues by going for the bleeding edge in our transpiling.